examples/regression/plot_timeseries_enbpi.py

```diff
@@ -0,0 +1,143 @@
+"""
+==================================================================
+Estimating prediction intervals of time series forecast with EnbPI
+==================================================================
+This example uses
+:class:`mapie.time_series_regression.MapieTimeSeriesRegressor` to estimate
+prediction intervals associated with time series forecast. The implementation
+is still at its first step, based on Jackknife+-after-bootsrtap, to estimate
+residuals and associated prediction intervals.
+
+We use here the Victoria electricity demand dataset used in the book
+"Forecasting: Principles and Practice" by R. J. Hyndman and G. Athanasopoulos.
+The electricity demand features daily and weekly seasonalities and is impacted
+by the temperature, considered here as a exogeneous variable.
+
+The data is modelled by a Random Forest model with a
+:class:`sklearn.model_selection.RandomizedSearchCV` using a sequential
+:class:`sklearn.model_selection.TimeSeriesSplit` cross validation, in which the
+training set is prior to the validation set.
+The best model is then feeded into
+:class:`mapie.time_series_regression.MapieTimeSeriesRegressor` to estimate the
+associated prediction intervals. We compare two approaches: one with no
+`partial_fit` call and one with `partial_fit` every 5 steps.
```

Comment:
Can you discuss the results between the two approaches ? What about their coverages and their mean widths ?